app/lib/actions/climbing/actions.ts

```diff
@@ -811,7 +826,7 @@ export async function deleteMemberCompletely(
     throw new Error('Erreur lors de la suppression du membre.');
   } finally {
     client.release();
-    revalidatePath('/dashboard/climbing');
+    //revalidatePath('/dashboard/climbing'); // A faire côté client car utilisé dans removeMemberFromSeason
```

Comment:
to remove si on n'en n'a pas besoin

Reply:
ok, fait. Pareil pour les console.warn et console.log